test-owasp.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
 
-zap-cli start --start-options '-config api.disablekey=true'
-zap-cli open-url $LIVE_TARGET
-zap-cli spider $LIVE_TARGET
-zap-cli active-scan -s all -r $LIVE_TARGET
-zap-cli alerts -l Informational -f json > results/zap.json
-zap-cli shutdown
-
-exit 0
+apt-get install jq
+pip install tinys3
+
+COUNTER=0
+COUNT=$(cat scripts/targets.json | jq '.targets[] .url' | wc -l)
+
+
+while [ $COUNTER -lt $COUNT ]; do
+  NAME=$(cat scripts/targets.json | jq ".targets[${COUNTER}] .name" -r)
+  TARGET=$(cat scripts/targets.json | jq ".targets[${COUNTER}] .url" -r)
+
+  echo Scanning $NAME: $TARGET
+  # Leaving ZAP up between runs allows alerts to accrue
+  zap-cli start --start-options '-config api.disablekey=true'
+  zap-cli open-url $TARGET
+  zap-cli spider $TARGET
+  zap-cli active-scan -s all -r $TARGET
+  zap-cli alerts -l Informational -f json > results/${NAME}.json
+  zap-cli shutdown
```

Comment:
Ah, hadn't realized that it was bundled with the ZAP Docker image, because it wasn't documented :unamused: Added!

https://github.com/zaproxy/zaproxy/wiki/Docker#zap-cli